api/users/users-router.js

```diff
@@ -21,7 +21,7 @@ router.post("/", async (req, res) => {
   try {
     const registered = await Users.add(user);
     console.log("registered: ", registered);
-    if (!registered.email && !registered.password) {
+    if (!registered.email || !registered.password) {
```

Comment:
this was the error in my logic that was making the error message appear, although the object was still being added to the database. this refers to issue #1